mitodl/bootcamp-ecommerce
# Profile adjustments

#### What are the relevant tickets?
Closes #460 

#### What's this PR do?
- Removes `leadership_level` from the `Profile` model
- Makes all profile form fields mandatory
- Moves some code from `../components/forms/ProfileFormFields` to `../constants` and `../lib/validation` (part of #471)

#### How should this be manually tested?
Log in and go to `../profile/edit`.  Make all fields blank and try to submit, you should see 'required' error messages for all fields, and should not be able to submit the form until all fields are populated.
